src/main/java/hudson/maven/RedeployPublisher.java

```diff
@@ -388,6 +388,7 @@ public boolean showEvenIfUnstableOption() {
     public static class WrappedArtifactRepository implements ArtifactRepository {
         private ArtifactRepository artifactRepository;
         private boolean uniqueVersion;
+        private boolean blocked;
```

Comment:
why is it needed?

Reply:
New API of the `ArtifactRepository` interface.